src/splitting.rs

```diff
@@ -36,6 +36,21 @@ impl WordSplitter for Box<dyn WordSplitter> {
         self.deref().split_points(word)
     }
 }
+
+impl WordSplitter for Box<dyn WordSplitter + Send> {
+    fn split_points(&self, word: &str) -> Vec<usize> {
+        use std::ops::Deref;
+        self.deref().split_points(word)
+    }
+}
+
+impl WordSplitter for Box<dyn WordSplitter + Send + Sync> {
+    fn split_points(&self, word: &str) -> Vec<usize> {
+        use std::ops::Deref;
+        self.deref().split_points(word)
+    }
+}
+
 /* Alternative, also adds impls for specific Box<S> i.e. Box<HyphenSplitter>
 impl<S: WordSplitter + ?Sized> WordSplitter for Box<S> {
```

Comment:
This should also solve it if I'm not mistaken and is more generic.

Reply:
This code is what I'm talking about.

Reply:
Sure, done :)